src/sql/workbench/api/common/extHostConnectionManagement.ts

```diff
@@ -78,6 +78,10 @@ export class ExtHostConnectionManagement extends ExtHostConnectionManagementShap
 		return this._proxy.$openChangePasswordDialog(profile);
 	}
 
+	public $openCustomErrorDialog(options: azdata.window.IErrorDialogOptions): Thenable<string | undefined> {
```

Comment:
I'd move this to extHostModelViewDialog since it is more related to this API.

Reply:
That looks like its working with `CustomDialogService` whereas I'm working with `ConnectionDialogService`

Reply:
I think this is better off in connection namespace, since the API applies specifically to ConnectionDialogService that shows error dialog for connection errors.

Reply:
it just happens to be used by connection scenario, but we might also use this to show error dialogs in other extensions.
also, the actual implementation is based on ErrorMessageService, you don't need to call all the way into connection management service.

Reply:
Addressed.